sample-plugin/typescript/src/index.ts

```diff
@@ -0,0 +1,31 @@
+// TIP: All user-defined code lives in the module folders (./query, ./mutation)
+
+import * as Internal from "./w3";
```

Comment:
I think we should just do the default `export * from "./w3"` method here, instead of overriding the plugin's constructor. This is something that shouldn't be necessary, and only used for "advanced" use cases.